message_ix/tests/test_tutorials.py

```diff
@@ -71,7 +71,7 @@ def _t(group: Union[str, None], basename: str, *, check=None, marks=None):
 #: Argument values to parametrize :func:`test_tutorial`.
 TUTORIALS: List[Tuple] = [
     # IPython kernel
-    _t("w0", f"{W}_baseline", check=[("solve-objective-value", 173795.09375)]),
+    _t("w0", f"{W}_baseline", check=[("solve-objective-value", 159025.82812)]),
```

Comment:
Can we explain precisely why the objective value has changed here?
Does this value still match the value of OBJ obtained from `.testing.make_westeros(…).solve()`?
If not, we need to clearly mark why (or perhaps adjust so the two still match), in order to avoid confusing ourselves in the future.

Reply:
@behnam-zakeri could know this, otherwise I will test once I find the time.

Reply:
Excellent question! I've made `make_westeros` consistent with the changes in the `westeros_baseline.ipynb`, which mainly meant introducing a technical lifetime and investment and fix costs for the grid. With these changes, loading a scenario from `make_westeros` and solving it yield `Objective:       159025.830480`, which is hopefully close enough. I'm not sure the values were exactly the same before.

Reply:
Should we bake this into a test requirement, that the `OBJ` value of a solved `make_westeros` needs to be consistent with the `OBJ` from the baseline tutorial? Maybe not with this PR, but in general this would expose tacit knowledge.

Reply:
Some other tests seem to rely on the exact values we use for `make_westeros`. How do we know it's safe to simply adapt the expected values?

Reply:
Thanks @khaeru for pointing to this and @glatterf42 for investigating. The objective value has changed, as noted, because of adding cost parameters for the "grid". Also, the input/output efficiency of the grid has been changed to be consistent with other technology, i.e., "output" has changed to 1 and inefficiencies are reflected in "input". I agree that the output from `make_westeros` should be the same as "westeros_basline.ipynb" to avoid any confusion in the future.

Reply:
All tests are passing again after I adapted the expected values, so please re-review, @khaeru. Not sure why the codecov checks fail (again).